release-notes/major12/12.0.0/valtimo-backend-libraries.md

```diff
@@ -75,6 +75,11 @@ The following features were added:
 
   Cases can be filtered on statuses set to the case via the process. More information on this can be found [here](/using-valtimo/case/internal-case-status.md)
 
+* **The RabbitMQ outbox publisher now support exchange targets**
+
+  Exchange targets can now be used with the outbox publisher for RabbitMQ by using the `valtimo.outbox.publisher.rabbitmq.exchange` property.
```

Comment:
Can you provide an example as well to show why this is useful?

Reply:
I added a bit about that.